src/styles/layout/_component-grid.scss

```diff
@@ -85,16 +78,25 @@ $breakpoint-accordion-large-755: 47rem;
     align-items: flex-start;
     color: var(--font-color-dark);
 
+    img {
+      object-position: bottom;
+      object-fit: scale-down;
+
+      @media (min-width: $breakpoint-scale-up) {
+        object-fit: contain;
+      }
+    }
```

Comment:
This looks awesome! I didn't know about the scale-down value, that's perfect for this situation. 

I'd appreciate if another dev would weigh in on this: should there be an @supports query wrapping object-fit? It [doesn't work in any versions of IE](https://caniuse.com/?search=object-fit), but at the same time, toggling object-fit at large screen sizes doesn't affect anything visually--it's really only important on mobile-sized screens that _probably_ won't be using IE.

If so, hiding the overflow for all the cards rather than just the accordion takes away any risk of it being a usability issue, even if it's not very pretty.

Reply:
Dustin said don't even worry about IE! Woo! 😮‍💨